README.md

```diff
@@ -50,10 +50,23 @@ Reminders and report_on_aftermath notifications are sent over email via `rake ma
 15 15 * * 1 bash -l -c "cd /path/to/eff_fab/ && /path/to/rake -t RAILS_ENV=production mail:send_last_minute_reminder"
 ```
 
-##### Customize the images
+### Customize the images
 
 Drop in more images into `app/assets/images/banner_pool` to have them show up in that beautiful top banner.  If one image doesn't seem to work, it probably has a `+` sign or possibly something even more adventurous like a question mark, so try taking that out.  
 
+Administering Users
+-------------------
+
+### From within the app
+
+1. Log in as a user with the admin role.
+2. Click "Administer" in the nav bar and click "Add new user".
+
+### From the API
+
+Like the app, the API uses basic auth for authentication. Along with your request, you must pass basic auth credentials associated with a user who has the admin role. In other words if the basic auth name is [NAME], there must be user with admin credentials and the email address [NAME]@eff.org.
```

Comment:
Keeping API access separate from basic auth sounds great to me! We should update this.